Add Quarkus-2394: Drop OpenTelemetry Jaeger exporter (#102)
quarkus-qe/quarkus-test-plans: QUARKUS-2394.md

# QUARKUS-2394 Drop OpenTelemetry Jaeger exporter

Jira: https://issues.redhat.com/browse/QUARKUS-2394

> Opentelemetry Jaeger exported has been deprecated from an official Quarkus support point of view. The extension `quarkus-opentelemetry-exporter-jaeger` will be moved to Quarkiverse and `quarkus-opentelemetry-exporter-otlp` extension has been promoted as the supported one. 

Reviewer: ```suggestion
OpenTelemetry Jaeger exporter has tech-preview support in RHBQ 2.7. Starting with RHBQ 2.13 the tech-preview support is dropped. The extension `quarkus-opentelemetry-exporter-jaeger` is planned to be moved to Quarkiverse in the future, `quarkus-opentelemetry-exporter-otlp` extension is the one to be used instead of Jaeger exporter. 
```

## Scope of the testing

> All references to `quarkus-opentelemetry-exporter-jaeger` are going to be removed. Current coverage are going to be replaced by `quarkus-opentelemetry-exporter-otlp` on OpenShift / Baremetal / Native

Reviewer: ```suggestion
All references to `quarkus-opentelemetry-exporter-jaeger` are going to be removed. Current coverage is going to be replaced by `quarkus-opentelemetry-exporter-otlp`.
```

> 

Reviewer: Add step to ensure metadata for TP are dropped

- update Marete ... https://gitlab.cee.redhat.com/rhoar-qe/marete/-/blob/main/src/main/resources/quarkus-2-13.yml#L282
  - that update needs to be done after the metadata changes are in
  - push on the JIRA to have the metadata update done
- you should check code.quarkus stage instance related to ER/CR build to visually ensure the TP label is gone
  - (after the metadata stuff is done and we have prod build with it)

Author: Done: https://gitlab.cee.redhat.com/rhoar-qe/marete/-/merge_requests/241

### Impact on test suites and testing automation

> The main impact are related to scenarios configuration and documentation in the following test suites:

Reviewer: ```suggestion
The main impact is related to scenarios configuration and documentation in the following test suites:
```

- Quarkus Test Suite

> - Beefy Scenarios

Reviewer: Drop BeefyTS, it's not actively maintained. We can keep the legacy exporter there.

Author: well...the PR is already done :(

Reviewer 2: Apologies.

- Start&Stop

### Impact on resources:

> - No additional requirements for resources in lab. Current coverage are going to be covered by `quarkus-opentelemetry-exporter-otlp` and the required resources are the same.

Reviewer: ```suggestion
- No additional requirements for resources in lab. Current coverage is going to be covered by `quarkus-opentelemetry-exporter-otlp` and the required resources are the same.
```

## Getting familiar with the feature

Following actions were taken to ensure familiarity:

> - Ensure documentation provides clear explanation on configuration options

Reviewer: upstream documentation ...

- Ensure good user experience and simplicity of use

## Advanced topics for test development

- No advanced topics found

## Contacts

* Tester: Pablo Gonzalez <pagonzal@redhat.com>

## References

> - [opentelemetry](https://quarkus.io/guides/opentelemetry)

Reviewer: ```suggestion
- [Quarkus OpenTelemetry guide](https://quarkus.io/guides/opentelemetry)
```

# QUARKUS-2394 Drop OpenTelemetry Jaeger exporter

Jira: https://issues.redhat.com/browse/QUARKUS-2394

OpenTelemetry Jaeger exporter has tech-preview support in RHBQ 2.7. Starting with RHBQ 2.13 the tech-preview support is dropped. The extension `quarkus-opentelemetry-exporter-jaeger` is planned to be moved to Quarkiverse in the future, `quarkus-opentelemetry-exporter-otlp` extension is the one to be used instead of Jaeger exporter.

## Scope of the testing

All references to `quarkus-opentelemetry-exporter-jaeger` are going to be removed. Current coverage is going to be replaced by `quarkus-opentelemetry-exporter-otlp`.

> Additionally we need to be sure that all metadata related to `quarkus-opentelemetry-exporter-jaeger` are dropped specially in [code.quarkus.io](https://code.quarkus.io/?extension-search=origin:platform%20quarkus-opentelemetry-exporter-jaeger)

Reviewer: https://code.quarkus.redhat.com/ ...

> Additionally we need to be sure that all metadata related to `quarkus-opentelemetry-exporter-jaeger` are dropped specially in [code.quarkus.io](https://code.quarkus.redhat.com/?extension-search=origin:platform%20quarkus-opentelemetry-exporter-jaeger)

Reviewer: ```suggestion
Additionally we need to be sure that all metadata related to `quarkus-opentelemetry-exporter-jaeger` are dropped especially in https://code.quarkus.redhat.com/?extension-search=origin:platform%20quarkus-opentelemetry-exporter-jaeger (to be checked on the stage instance)
```
